src/Nethermind/Nethermind.Evm/TransactionProcessing/TransactionProcessor.cs

```diff
@@ -343,7 +344,7 @@ protected TransactionResult ValidateStatic(
                 return "EIP-3860 - transaction size over max init code size";
             }
 
-            return ValidateGas(tx, header, intrinsicGas, validate);
+            return ValidateGas(tx, header, Math.Max(intrinsicGas, floorGas), validate);
```

Comment:
Oh I see it is needed here. Perhaps, out is the best solution then, but think about it